docs/release_notes.md

```diff
@@ -1,5 +1,160 @@
 #
 
+## 5.0.0
+
+Release Date: 2025-05-XX
+
+The iRODS Consortium and RENCI are pleased to announce iRODS 5.0.0.
+
+This release represents a significant effort to make iRODS what users expect from an enterprise grade data management platform. Great focus has been put into making iRODS 5 easy to deploy and manage in a variety of environments.
+
+Notable updates include:
+
+- **Revamped Startup/Shutdown Process** - The startup/shutdown process has been fully redesigned. The server honors signals in the expected manner and also communicates its status to the service manager. Administrators will find that iRODS 5 is significantly easier to deploy in the cloud, containers, you name it.
+- **Unified Server Configuration** - All server configuration options, excluding zone-wide options, have been consolidated into `server_config.json`. The server no longer relies on the service account's `irods_environment.json` file.
```

Comment:
Other instances of server_config.json and irods_environment.json aren't wrapped in backticks.

Be consistent.

Reply:
Done.